gpbft/chain_test.go

```diff
@@ -7,56 +7,17 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func TestTipSet(t *testing.T) {
```

Comment:
This test really wasn't doing much.

Reply:
I'm happy to add something back to test `IsZero`, `Equals` and the stringer if desired.

Reply:
Please feel free to remove the test. 

(not now) At some point we should review the exported `gpbft` package APIs and avoid having multiple ways of doing the same thing. Which I suspect might be the root cause.